server/src/main/java/org/elasticsearch/cluster/metadata/NodesShutdownMetadata.java

```diff
@@ -67,27 +67,6 @@ public static NamedDiff<Metadata.Custom> readDiffFrom(StreamInput in) throws IOE
         return new NodeShutdownMetadataDiff(in);
     }
 
-    public static Optional<NodesShutdownMetadata> getShutdowns(final ClusterState state) {
-        assert state != null : "cluster state should never be null";
-        return Optional.of(state).map(ClusterState::metadata).map(m -> m.custom(TYPE));
-    }
-
-    /**
-     * Returns true if the given node is marked as shutting down with any
-     * shutdown type.
-     */
-    public static boolean isNodeShuttingDown(final ClusterState state, final String nodeId) {
-        // Right now we make no distinction between the type of shutdown, but maybe in the future we might?
-        return NodesShutdownMetadata.getShutdowns(state)
-            .map(NodesShutdownMetadata::getAllNodeMetadataMap)
-            .map(allNodes -> allNodes.get(nodeId))
-            .isPresent();
-    }
-
-    public static NodesShutdownMetadata getShutdownsOrEmpty(final ClusterState state) {
-        return getShutdowns(state).orElse(EMPTY);
-    }
```

Comment:
This is replaced with `org.elasticsearch.cluster.metadata.Metadata#nodeShutdowns` that never returns null